src/components/organization.astro

```diff
@@ -50,6 +50,13 @@ const features = [
     photo: "/images/org/cristian.jpg",
     url: "https://maureira.xyz",
   },
+  {
+    name: "Laís",
+    role: "Sponsorship Co-Chair | Conference Co-Chair",
```

Comment:
```suggestion
    role: "Accessibility Chair",
```
I think you copied some other from the top, so would you be OK if we settle with the Accessibility one? I know it's not defined yet, but it's what you mentioned had interest with.